core/dbt/cli/flags.py

```diff
@@ -32,13 +33,20 @@ def assign_params(ctx):
 
         assign_params(ctx)
 
+        # Get the invoked command flags
```

Comment:
FYI-- I reworked this to be [a lot cleaner and remove some potential edge cases.](https://github.com/dbt-labs/dbt-core/blob/c3ea0d1e01adc70d000375fb73b9897ffe45ec4d/core/dbt/cli/flags.py#L38)  No need to fix it now unless you want to since it's part of the logging branch I'm working on, but worth noting.